### PR TITLE
Feat/support more queries

### PR DESCRIPTION
**Pull request by lamcc21**
TODO:
- [x] add config for custom scalars
- [x] fix random issue with codegen for 2 types (lockedAmount & withdrawls)
- [x] bump apollo/client version and add dotenv for codegen.yaml
- [x] Script for export file and add tests

### CODE REVIEW

File: packages/network-clients/src/clients/queryClient.ts [hunk 1]
```diff
@@ -36,9 +42,9 @@ export class GraphqlQueryClient {
 
   // QUERY REGISTRY QUERY FUNCTIONS
 
-  async getIndexer(address: string): Promise<GetIndexer_indexer> {
+  async getIndexer(address: string): Promise<any> {
     const result = await wrapApolloResult(
-      this.explorerClient.query<GetIndexer>({
+      this.explorerClient.query<GetIndexerQuery, GetIndexerQueryVariables>({
```
Comment (CaiYiLiang) [inline packages/network-clients/src/clients/queryClient.ts:47]: Should also support all options supported by useQuery, for exmaple: `pollInterval`

Comment (lamcc21) [reply packages/network-clients/src/clients/queryClient.ts:47]: This query function will only be used internally in the package. You will be able to import useQuery hook (from react-hooks) that supports pollInterval automatically.

File: packages/network-clients/codegen.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+schema: 'https://api.subquery.network/sq/subquery/kepler-testnet-subql-project'
```
Comment (HZHAIX) [inline packages/network-clients/codegen.yaml:1]: make the schema link an env so that it can be switch when user connect with different network

**Comment by HZHAIX** [comment]
Awesome work! Would be good to have a simple test for testing a query client. 
https://github.com/subquery/network-clients/blob/main/test/client.test.ts

We can add more test for all the queries in the next stage

File: packages/network-clients/codegen.yaml (hunk 2 above)
Comment (lamcc21) [reply packages/network-clients/codegen.yaml]: done

File: test/queryClient.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  beforeAll(async () => {
+    const config = NETWORK_CONFIGS.kepler;
+    assert(config, 'network config not defined');
+    client = new GraphqlQueryClient(config);
+  }, 160000);
+
+  it('can query indexer detail', async () => {
+    const apolloClient = client.explorerClient;
+    const result = await apolloClient.query({
```
Comment (HZHAIX) [inline test/queryClient.test.ts:20]: Try to destruct the `result`, and check each of them, the value can not be undefined:
```
const { 
   id,
    metadata,
    controller,
    commission,
    totalStake
}  = result;
```

**Pull request merged**